Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -170,7 +183,38 @@ open class ComposerVC: _ViewController,
                 attachments: [],
                 mentionedUsers: mentionedUsers,
                 command: command,
-                extraData: extraData
+                extraData: extraData,
+                cooldownTime: cooldownTime
+            )
+        }
+        
+        public mutating func slowdownMode(cooldown: Int) {
+            self = .init(
+                text: text,
+                state: state,
+                editingMessage: editingMessage,
+                quotingMessage: quotingMessage,
+                threadMessage: threadMessage,
+                attachments: attachments,
+                mentionedUsers: mentionedUsers,
+                command: command,
+                extraData: extraData,
+                cooldownTime: cooldown
+            )
+        }
+        
+        public mutating func resetSlowMode() {
+            self = .init(
+                text: text,
+                state: state,
+                editingMessage: editingMessage,
+                quotingMessage: quotingMessage,
+                threadMessage: threadMessage,
+                attachments: attachments,
+                mentionedUsers: mentionedUsers,
+                command: command,
+                extraData: extraData,
+                cooldownTime: 0
             )
```

Comment:
is it equivalent to?
```suggestion
            slowMode(cooldown: 0)
```